README.md

```diff
@@ -14,7 +14,7 @@ uncontainerized local versions of the dependencies too.
 
 ### Virtual Environment
 
-Create and activate a virtual environment with Python 3.7+.  
```

Comment:
Two trailing spaces in Markdown carry semantic meaning (simple line breaks) – could they be spared from linting? Maybe we exclude Markdown files from linting altogether? (Although removing the single trailing spaces in line 3/4 is good...)

Reply:
I think that is actually my editor, not the linting... Let me double check.